io/include/pcl/io/hdl_grabber.h

```diff
@@ -221,24 +226,24 @@ namespace pcl
 #pragma pack(push, 1)
       typedef struct HDLLaserReturn
       {
-          unsigned short distance;
-          unsigned char intensity;
+          uint16_t distance;
+          uint8_t intensity;
       } HDLLaserReturn;
 #pragma pack(pop)
 
       struct HDLFiringData
       {
-          unsigned short blockIdentifier;
-          unsigned short rotationalPosition;
+          uint16_t blockIdentifier;
+          uint16_t rotationalPosition;
           HDLLaserReturn laserReturns[HDL_LASER_PER_FIRING];
       };
 
       struct HDLDataPacket
       {
           HDLFiringData firingData[HDL_FIRING_PER_PKT];
-          unsigned int gpsTimestamp;
-          unsigned char mode;
-          unsigned char sensorType;
+          uint32_t gpsTimestamp;
```

Comment:
It might be better to bump this to 64 bits.

Reply:
In the manual, GPS time stamp is 4 bytes (32 bits).
Why bump to 64 bits?

Reply:
I just assumed it was gonna be a big number. 32 bits it is then.

Reply:
I will adopt 32 bits according to the manual. (It is the same bit width as before.)